Userland/Libraries/LibWeb/XHR/XMLHttpRequest.cpp

```diff
@@ -426,6 +428,21 @@ static bool is_header_value(String const& header_value)
     return true;
 }
 
+static XMLHttpRequest::BodyWithType safely_extract_body(XMLHttpRequestBodyInit& body)
```

Comment:
but can _this_ be a const reference?

Reply:
It could if this was const, I think: https://github.com/SerenityOS/serenity/blob/6978b53ea00dfa731a0ead82832f59069410108a/Userland/Libraries/LibWeb/URL/URLSearchParams.h#L44

Reply:
:yakplus: